Source/WebCore/css/CSSProperties.json

```
@@ -7284,7 +7284,8 @@
                 "start",
                 "end",
                 "self-start",
-                "self-end"
+                "self-end",
+                "anchor-center"
```

Presumably, you want this:

```suggestion
                {
                    "value": "anchor-center",
                    "settings-flag": "cssAnchorPositioningEnabled"
                }
```